# <center><u>***PROJECT-10***</u></center>
# <center><u>***Mean Yield of Catalyst***</u></center>

**For a chemical plant, two catalysts have been analyzed to test the mean yield of a chemical process. The catalyst 1 is currently in use but the process engineer wants to test whether catalyst 2, that is cheaper, could be adopted. The test is conducted in the pilot plant and the mean yield for each catalyst is attached.**

***Is there any difference in mean yield. Use α = 0.05 and assume equal variance?***

**1) First load the data in pandas.**

In [3]:
import pandas as pd
data=pd.read_csv('catalyst.csv')
data

Catalyst1  Catalyst2
0      91.50      89.19
1      94.18      90.95
2      92.18      90.46
3      95.39      93.21
4      91.79      97.19
5      89.07      97.04
6      94.72      91.07
7      89.21      92.75

**2) Calculate the sample mean and sample standard deviation for both the catalyst.**

In [16]:
print('The mean of \'Catalyst 1\' is:',data['Catalyst1'].mean())
print('The standard deviation of \'Catalyst 1\' is:',data['Catalyst1'].std())

The mean of 'Catalyst 1' is: 92.25500000000001
The standard deviation of 'Catalyst 1' is: 2.385018718105646


In [17]:
print('The mean of \'Catalyst 2\' is:',data['Catalyst2'].mean())
print('The standard deviation of \'Catalyst 2\' is:',data['Catalyst2'].std())

The mean of 'Catalyst 2' is: 92.73249999999999
The standard deviation of 'Catalyst 2' is: 2.983453176629871


**3) The number of rows for each catalyst.**

In [28]:
print('The number of rows for \'Catalyst 1\' are :',data['Catalyst1'].count(),'\nThe number of rows for \'Catalyst 2\' are:',data['Catalyst2'].count())

The number of rows for 'Catalyst 1' are : 8 
The number of rows for 'Catalyst 2' are: 8


**4) Use the eight step hypothesis process.**

<u>***STEP-1:***</u>  The parameter of interest are $μ_1$ and $μ_2$, the mean process yield using catalyst 1 and 2 respectively.<br>We want to know if $μ_1$ - $μ_2$=0.

<u>***STEP-2:***</u>  Null Hypothesis $H_0$ : $μ_1$ - $μ_2$=0, or $H_0$: $μ_1$=$μ_2$.

<u>***STEP-3:***</u>  Alternative Hypothesis $H_α$ : $μ_1$ ≠ $μ_2$.

<u>***STEP-4:***</u>  α = 0.05

<u>***STEP-5:***</u>  We need to calculate the test statistic with the help of $t_0$.

<u>***STEP-6:***</u>  Reject $H_0$ if $t_0$ > t <sub>0.025,14</sub> = 2.145 or if $t_0$ < -t <sub>0.025,14</sub> = - 2.145

In [30]:
from scipy.stats import t
t.ppf(1.95/2,14)

2.1447866879169273

<u>***STEP-7:***</u>  
**Computation:** we have $\bar x_1$ = 92.55, $s_1$ = 2.39, $n_1$ = 8, $\bar x_2$ = 92.733, $s_2$ = 2.98, $n_2$ = 8  
$s_p$ = $\sqrt{7.30}$ = 2.70

<u>***STEP-8:***</u>  
**Conclusions:** Since -2.145 < $t_0$ = -0.35 < 2.145 , the null hypothesis cannot be rejected. That is at the 0.05 significance level, we do not have strong evidence to conclude that catalyst 2 results in a mean yield that differs from the mean yield when catalyst 1 is used.<br>

***Alternatively, we can use scipy.stats, a submodule of scipy***

In [32]:
import scipy.stats as st
st.ttest_ind(data['Catalyst1'],data['Catalyst2'])

Ttest_indResult(statistic=-0.3535908643461798, pvalue=0.7289136186068217)

The method returns the ***t*** statistic and ***p*** value